12/umbraco-cms/tutorials/creating-a-property-editor/part-2.md

```diff
@@ -20,7 +20,7 @@ To add configuration options to our Suggestion Data Type, open the `package.mani
 ...
 "editor": {
                 "view": "~/App_Plugins/Suggestions/suggestion.html"
-            }, // Remeber a comma seperator here at the end of the editor block!
+            }, // Remember a comma separator here at the end of the editor block!
```

Comment:
to keep the docs consistent, can you please make this change to v10 and v11 as well?